README.md

```diff
@@ -24,10 +24,13 @@ You need to provide prebuilds made by [prebuild](https://github.com/mafintosh/pr
 prebuild-install [options]
 
   --download    -d  [url]       (download prebuilds, no url means github)
-  --no-prebuild                 (skip prebuild download)
```

Comment:
Recommend `--build-from-source` instead since `--prebuild` has a different behavior in `prebuild`.

`--no-prebuild` will still work: https://github.com/mafintosh/prebuild-install/pull/19/commits/d6e6d7e196a99e3ad86ee0fdcd31974a91aa8451#diff-13198ad72a3472842248bc3d0f0d3fd7R50